tests/phpunit/tests/block-template.php

```diff
@@ -206,6 +207,8 @@ public function test_get_the_block_template_html_enforces_singular_query_loop()
 		$wp_query     = new WP_Query( array( 'p' => $post_id ) );
 		$wp_the_query = $wp_query;
 
+		// Force a template ID that is for the current stylesheet.
+		$_wp_current_template_id = get_stylesheet() . '//single';
```

Comment:
I know the query globals are reset by the base `set_up()`, but shouldn't `$_wp_current_template_id` be unset in this class' `tear_down()`, just as `$_wp_current_template_content` is?

Reply:
@costdev In tests/phpunit/tests/media.php we don't unset `$_wp_current_template_content` in `tear_down()` 🤔

Reply:
@mukeshpanchal27 If (1) it's set by tests in that file, and (2) it spills into other tests, then it should be handled after each test completes. i.e. unset if it didn't exist, or reset to the previous value if it did exist.

Reply:
@costdev Agreed, I'll clear up these globals.